src/core/Akka.FSharp/FsApi.fs

```diff
@@ -37,7 +37,7 @@ module Serialization =
                         
     let internal exprSerializationSupport (system: ActorSystem) =
         let serializer = ExprSerializer(system :?> ExtendedActorSystem)
-        system.Serialization.AddSerializer(serializer)
+        system.Serialization.AddSerializer("expr", serializer)
```

Comment:
So "expr" is in the manifest type for the expression serializer?

Reply:
https://github.com/akkadotnet/akka.net/blob/55388c672d9fc9508c7e79231114de58a230faed/src/core/Akka/Serialization/Serialization.cs#L74  Just find one usage of this API here, I figured this might be "expr", Any suggestion? I will fix it